src/plugin.js

```diff
@@ -90,7 +90,8 @@ function dispatchEvent(chart, listeners, label) {
   }
 }
 
-function dispatchMoveEvents(chart, listeners, previous, label) {
+function dispatchMoveEvents(chart, listeners, labels, event) {
```

Comment:
What's the reason for nesting `previous` and `label` under `labels`? If it's to avoid the "too many arguments" ESLint error, I would instead disable ESLint for this line and avoid impacting the runtime.

Reply:
that's the reason. I'll change it

Reply:
done

Reply:
Now I remember why I did in that way. It's not LINT but CC. See CC issue https://codeclimate.com/github/chartjs/chartjs-plugin-datalabels/pull/271

Reply:
Ok, let's update the [CC config](https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/.codeclimate.yml#L2) then to increase the threshold:

```yaml
checks:
  argument-count:
    enabled: true
    config:
      threshold: 5
  method-complexity:
    enabled: false
  ...
```

Reply:
done